src/runner.py

```diff
@@ -289,6 +368,9 @@ def post_to_onboarding_lights(self, data):
                     url=ONBOARDING_LEDS_API,
                     json=data,
                     timeout=5,
+                    headers={
```

Comment:
Think you'll also need: `global onboarding_authentication_token  # pylint: disable=global-statement` in this method.